content/docs/reference/design-docs/timetable/index.en.md

```diff
@@ -158,8 +158,8 @@ margins:
   # the following units are supported:
   #  - none is a special value which indicates no margin. It's the default
   #  - % means added percentage of the base simulation time
-  #  - min/km means minutes per kilometers
-  values: ["5%", "3%"]
+  #  - min/100km means minutes per 100 kilometers
+  values: ["5%", "3%", "4,5min/100km"]
```

Comment:
In english the comma is a thousands separator, not the decimal place separator

Reply:
Done, thanks.